test-cases/manager/manager-restore-outside-with-las-and-primary-replica-only.yaml

```diff
@@ -0,0 +1,14 @@
+test_duration: 600
```

Comment:
Seems like a dup of the second one with just 1-2 additional yamls, so they can share the same yaml and this one will be additional configuration on top of it.
It makes it easier when you want to do the same change in both tests in order to do comparisons.

Reply:
make sense, will fix it